FL/Fl_Window.H

```diff
@@ -351,6 +351,7 @@ public:
   // These 2 member functions break the driver model but are kept for back compatibility.
   // They are implemented in Fl_win32.cxx
   static void default_icons(HICON big_icon, HICON small_icon);
+  static void default_icons();
   void icons(HICON big_icon, HICON small_icon);
```

Comment:
It is clearly said in the comment above: "These 2 member functions break the driver model but are kept for back compatibility."
Changing 2 to 3 could NOT be accepted, we should definitely not add new methods to those "kept for back compatibility".